kafka_exporter.go

```diff
@@ -384,7 +394,32 @@ func (e *Exporter) collect(ch chan<- prometheus.Metric) {
 
 	topicChannel := make(chan string)
 
-	getTopicMetrics := func(topic string) {
+	// get topic details
+	aTopics, err := e.admin.ListTopics()
+	if err != nil {
+		panic(err)
+	}
+
+	cfg, err := e.admin.DescribeConfig(sarama.ConfigResource{
+		Type: sarama.TopicResource,
+		Name: "min.insync.replicas",
+	})
+
+	// get min.insync.replicas from global config
+	defaultMinInsyncReplicas := float64(-1)
+	for _, v := range cfg {
+		if v.Name == "min.insync.replicas" {
+			minIsr, err := strconv.ParseFloat(v.Value, 64)
+			if err != nil {
+				// something went wrong : don't set it
+				minIsr = -1
+			} else {
```

Comment:
the `else` is not necessary here, right ?

Reply:
true.